learn/module_1.py

```diff
@@ -93,7 +93,22 @@ def test_fix_me_Marty_returns_hello_Marty(self):
 
     # Write a test that checks that fix_me returns a string.
     # def put_the_test_here!
-
+    def test_fix_me_returns_a_string(self):
+        """Check that fix_me() returns a string
+        """
+        some_str = 'string'
+        self.addTypeEqualityFunc(str, fix_me(some_str))
+        
```

Comment:
This first function isn't testing an assertion. It is meant for registering a new function that you can use with `assertEqual`.  You might use it if you were creating a new type.